noisecapture.html

```diff
@@ -169,11 +192,45 @@ <h2 class="centered">NoiseCapture in video</h2>
 					</p>
                 </div><!-- col-lg-4 -->
 
+                <!-- NoiseCapture resources -->
+                <div class="col-lg-4 callout">
+                    <i class="fa fa-code fa-4x" aria-hidden="true"></i>
+                    <h2>Resources</h2>
+                    <p align="justify">
+                        - Download NoiseCapture on the <a href="https://play.google.com/store/apps/details?id=org.noise_planet.noisecapture" target="blank">Google Play</a> store.
+                    <br>
+                        - Have a look to our <a href="https://github.com/Ifsttar/NoiseCapture" target="blank">GitHub code repository</a> to contribute to NoiseCapture's devlopments. 
+                    </p>
+                </div><!-- col-lg-4 -->
+
 			</div><!-- row -->
 
         </div><!-- Container -->
         <br>
 
+<!------------------------------------------------------------------------------------------------------------------------------------------------------------------>
+<!-- ==== NoiseCapture contributors ==== -->
+            <div class="container" id="Noisecapture_contrib" name="Noisecapture_contrib">
+
+             <div class="row">
+                <h2 class="centered">NoiseCapture contributors</h2>
+                <hr>
+                <br>
+                <div class="col-lg-offset-2 col-lg-8">
+                    <p align="justify">
+                    Special thanks to the following contributors, who helped improve NoiseCapture.
```

Comment:
We must list here only translations. Otherwise we will have a very large and never updated list (ie a developer who proposes a new feature ).
What do u think ?